topbeat/beat/sigar.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/elastic/gosigar"
+	"github.com/eonarheim/gosigar"
```

Comment:
It would be good if you could open first a PR to github.com/elastic/gosigar with your changes from github.com/eonarheim/gosigar so we don't have to change the full dependency.